docs: move traefik service configuration to docker (#2)
RealOrangeOne/traefik-pages: README.md

      - SITES_ROOT=/mnt/sites
      - TRAEFIK_SERVICE=traefik-pages@docker
      - AUTH_PASSWORD=hunter2
    labels:
      - traefik.enable=true

>       - traefik.http.services.traefik-pages.loadbalancer.server.port=5000

Reviewer: This line shouldn't be necessary, as Traefik will identify the port automatically through the `EXPOSE` in the `Dockerfile`.

```

This doesn't need to be in the same file as Traefik, but it does need to be accessible to Traefik using a fixed hostname and IP. If Traefik is running in host mode (as I do), you'll need to bind `traefik-pages` to an internal interface, and listen to that.

> The labels enable traefik autoconfiguration and configure the service listener. Note that the service name should match `$TRAEFIK_SERVICE`. The second label can be omitted, but $TRAEFIK_SERVICE will need to match the automatically configured name of the service.

Reviewer: ```suggestion
The label enables traefik autoconfiguration to detect `traefik-pages`. Note that `$TRAEFIK_SERVICE` must match the service name created by traefik. 
```

Author: Thanks, I couldn't figure out how to word that in a way that made sense.
